packages/utils/src/hex_utils.ts

```diff
@@ -108,3 +112,10 @@ function toHex(n: Numberish, _size: number = WORD_LENGTH): string {
     }
     return `0x${_n.toString(16)}`;
 }
+
+/**
+ * Check if a string is a hex string.
+ */
+function isHex(s: string): boolean {
```

Comment:
Nit could be good to have `A-F` as well. Especially when working w addresses.

Reply:
Should be case-insensitive with the `i` flag.